ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
+    > random(1000)
+    < 882.4784
+    ~ \d+\.\d+
     > sum(1..1000, random(5))
-    < 2464.8003929607607
+    < 2464.8004
     ~ 2\d\d\d\.\d+
```

Comment:
According to [my computations](http://www.wolframalpha.com/input/?i=1%2Berf%28%282000-2500%29%2Fsqrt%282*5*1000%2F12%29%29) we're talking of a probability of 1.7<sup>−132</sup> here. I'd say we can safely ignore that problem, since there are unavoidable technical reasons why Travis might fail which are far more likely.